lib/migrator.js

```diff
@@ -31,6 +31,11 @@ module.exports = async function(args) {
   let currentPosts = [];
 
   const md = str => {
+    // #35
+    if (!/<p>/i.test(str) && !/<code>/i.test(str)) {
```

Comment:
`!/<code>/i.test(str)` is to avoid messing around with codeblock.

I don't know how WP formats the codeblock though, maybe it only uses `<pre>` or `<table>`. @adnan360 @jehy can I request a sample xml of inline code (e.g. "lorem `ipsum` dolor") and codeblock in both classic and modern (guternberg) WP editor?

Reply:
https://github.com/hexojs/hexo-migrator-wordpress/pull/79#issuecomment-664820141